lib/shopify_api/rest/base.rb

```diff
@@ -313,6 +328,23 @@ def save(update_object: false)
         @errors.errors << e
         raise
       end
+
+      private
+
+      sig { params(var: T.any(String, Symbol), val: T.untyped).void }
+      def set_property(var, val)
+        clean = var.to_s.gsub(/[\?\s]/, "")
+        @aliased_properties[var.to_s] = clean if clean != var
```

Comment:
Some of the fields returned by the API have special characters in them - I don't love this solution, but I couldn't figure out a better way of cleaning up the instance attributes.